CommandLine.Tests/ParserTests.cs

```diff
@@ -251,6 +251,45 @@ public void Option_short_forms_can_be_bundled()
         }
 
         [Fact]
+        public void Options_short_forms_do_not_get_unbundled_if_unbundling_is_turned_off()
+        {
+            var parser = new Parser(
+                false,
+                Command("the-command", "",
+                        Option("-x", "", NoArguments()),
+                        Option("-y", "", NoArguments()),
+                        Option("-z", "", NoArguments()),
+                        Option("-xyz", "", NoArguments())));
+
+            var result = parser.Parse("the-command -xyz");
+
+            result["the-command"]
+                .AppliedOptions
+                .Select(o => o.Name)
+                .Should()
+                .BeEquivalentTo("xyz");
+        }
+
+        [Fact]
+        public void Option_short_forms_get_unbundled_if_unbundling_is_explicitly_turned_on()
```

Comment:
It looks equivalent to https://github.com/dotnet/CliCommandLineParser/pull/58/files#diff-29b1f4eae325182f6805bdf86bbf4fd7R236.